diff --git a/sass.dart.js b/sass.dart.js
index 48e85223a21c608ef8d3b2903ed8687bd98d0ff4..95336a6d5bbeace12f0e3fb175fbc6f49d50cedf 100644
--- a/sass.dart.js
+++ b/sass.dart.js
@@ -21478,6 +21478,14 @@ self.util = _cliPkgRequires.util;
           A.throwExpression(A.ArgumentError$(string$.parsed, null));
         else {
           t2 = value.value;
+          if (
+            t2 instanceof A.SassList0 ||
+            t2 instanceof A.SassColor0 ||
+            t2 instanceof A.UnitlessSassNumber0 ||
+            t2 instanceof A.SingleUnitSassNumber0
+          ) {
+            t2 = value.value = new A.SassString0(t2.toString$0(0), false, value.span);
+          }
           if (!(t2 instanceof A.SassString0))
             A.throwExpression(A.ArgumentError$(string$.If_par + value.toString$0(0) + "` of type " + A.getRuntimeTypeOfDartObject(t2).toString$0(0) + ").", null));
         }
